src/utils/cookie.js

```diff
@@ -0,0 +1,38 @@
+import { fs } from './fs';
+
+/**
+ * Parse the available cookies from the document.cookie variable
+ * @returns {{}} An object with key/value pairs of cookie.name=cookie.value
+ */
+export function getAvailableCookies() {
+  let cookies = {};
+  let cookieParts = document.cookie.split(';');
+  for( let i=0; i<cookieParts.length; i++ ){
+    let cookieSplit = cookieParts[i].split('=');
+    if( cookieSplit.length === 2 ){
+      cookies[cookieSplit[0].trim()] = cookieSplit[1];
+    }
+  }
+  return cookies;
+}
+
+/**
+ * Set fs userVars based on cookie names.  Will be set as cookie.name=cookie.value
+ * If cookie is not available, it will not be in the userVars, and a warning will be printed
+ * Also note that userVars in FullStory cannot start with _ character, so make sure your cookies
+ * don't start with that otherwise they won't appear.
+ * @param cookieNames An array of cookie names to set as values
+ */
+export function setCookiesAsUserVars( cookieNames ){
```

Comment:
I lean to this being more of the actual logic of the example.  The cookie utility could help get/set cookies, but doing something with the cookie value itself seems delegated to the example script(s).

Reply:
That is fine.  I was torn on that.  Wasn't sure if other scripts might want to also set cookies as userVars, but if that happens they can move the code at that time.  I'll move that method into the example script.